src/core/nginx.go

```diff
@@ -810,13 +810,16 @@ func runtimeFromConfigure(configure []string) []string {
 // AccessLogs returns a list of access logs in the config
 func AccessLogs(p *proto.NginxConfig) map[string]string {
 	var found = make(map[string]string)
+	var name := ""
```

Comment:
Both of the following are equivalent and should satisfy the linter:
```suggestion
	name := ""
```
or
```suggestion
	var name
```